client/landing/gutenboarding/path.ts

```diff
@@ -97,8 +101,30 @@ export function useNewQueryParam() {
 	return new URLSearchParams( useLocation().search ).has( 'new' );
 }
 
+export function useIsAnchorFm(): boolean {
+	const podcastId = useAnchorFmPodcastId();
+	return podcastId !== null && podcastId !== '';
+}
+
+// Returns the anchor podcast id. First looks in "location state",
+// provided by react-router-dom, if not available there, checks the query string.
+export function useAnchorFmPodcastId(): string | null {
+	const { state } = useLocation< GutenLocationStateType >();
```

Comment:
Should we call `state` here something like `locationState` to differentiate it from Redux state?

Reply:
sure, I can rename state to locationState most everywhere: [fix/anchor-gutenboarding-f5 6b5dd95ee6] Rename state to locationState